sherpa/csrc/rnnt_emformer_model.cc

```diff
@@ -53,7 +53,7 @@ RnntEmformerModel::RnntEmformerModel(const std::string &filename,
   right_context_length_ = encoder_.attr("right_context_length").toInt();
 }
 
-std::pair<torch::Tensor, RnntEmformerModel::State>
+std::tuple<torch::Tensor, torch::Tensor, RnntEmformerModel::State>
```

Comment:
For the following CI error
https://github.com/k2-fsa/sherpa/runs/7444603197?check_suite_focus=true

You can check the style of the C++ code locally by
```bash
./scripts/check_style_cpplint.sh 2
```

Reply:
Yup, already working on it ;)